diff -ru netcdf-4.3.1.orig/include/netcdf.h netcdf-4.3.1/include/netcdf.h
--- netcdf-4.3.1.orig/include/netcdf.h	2014-01-16 15:09:24.000000000 -0700
+++ netcdf-4.3.1/include/netcdf.h	2014-10-22 08:58:26.687510896 -0600
@@ -225,11 +225,11 @@
 As a rule, NC_MAX_VAR_DIMS <= NC_MAX_DIMS.
 */
 /**@{*/
-#define NC_MAX_DIMS	1024	
+#define NC_MAX_DIMS	65536	
 #define NC_MAX_ATTRS	8192	
-#define NC_MAX_VARS	8192	
+#define NC_MAX_VARS	524288	
 #define NC_MAX_NAME	256	
-#define NC_MAX_VAR_DIMS	1024 /**< max per variable dimensions */
+#define NC_MAX_VAR_DIMS	8 /**< max per variable dimensions */
 /**@}*/
 
 /** This is the max size of an SD dataset name in HDF4 (from HDF4 documentation).*/
