diff -ru netcdf-4.1.1.orig/libsrc/netcdf.h netcdf-4.1.1/libsrc/netcdf.h
--- netcdf-4.1.1.orig/libsrc/netcdf.h	2010-01-12 08:44:57.000000000 -0700
+++ netcdf-4.1.1/libsrc/netcdf.h	2010-11-17 10:02:18.892780736 -0700
@@ -141,11 +141,11 @@
  * applications and utilities.  However, nothing is statically allocated to
  * these sizes internally.
  */
-#define NC_MAX_DIMS	1024	 /* max dimensions per file */
+#define NC_MAX_DIMS	65536    /* max dimensions per file */
 #define NC_MAX_ATTRS	8192	 /* max global or per variable attributes */
-#define NC_MAX_VARS	8192	 /* max variables per file */
+#define NC_MAX_VARS	524288   /* max variables per file */
 #define NC_MAX_NAME	256	 /* max length of a name */
-#define NC_MAX_VAR_DIMS	NC_MAX_DIMS /* max per variable dimensions */
+#define NC_MAX_VAR_DIMS	8        /* max per variable dimensions */
 
 /*
  * The netcdf version 3 functions all return integer error status.
Only in netcdf-4.1.1/libsrc: netcdf.h.orig
